core/indexer/interface.go

```diff
@@ -1,24 +1,13 @@
 package indexer
 
 import (
-	"bytes"
-	"math/big"
-
-	"github.com/ElrondNetwork/elrond-go/core/indexer/workItems"
+	"github.com/ElrondNetwork/elastic-indexer-go/workItems"
 	"github.com/ElrondNetwork/elrond-go/core/statistics"
 	"github.com/ElrondNetwork/elrond-go/data"
-	"github.com/ElrondNetwork/elrond-go/data/block"
 	"github.com/ElrondNetwork/elrond-go/data/state"
 	"github.com/ElrondNetwork/elrond-go/process"
-	"github.com/elastic/go-elasticsearch/v7/esapi"
 )
 
-// DataIndexerFactory can create new instances of Indexer
-type DataIndexerFactory interface {
-	Create() (Indexer, error)
-	IsInterfaceNil() bool
-}
-
 // Indexer is an interface for saving node specific data to other storage.
 // This could be an elastic search index, a MySql database or any other external services.
 type Indexer interface {
```

Comment:
Can we move this only when it is used? Or directly in core package?

Reply:
I kept the same package because If I will move the interface I will also have to change it all over the places where is used. If you strongly suggest moving the interface a PR is required also on the elastic-indexer-go repository.

Reply:
Maybe write a TODO to come back later and clean the core/indexer package? Or better: a Jira task?

Reply:
Done. The interface was moved.